src/AppBundle/Controller/BlogController.php

```diff
@@ -114,4 +116,43 @@ public function commentFormAction(Post $post)
             'form' => $form->createView(),
         ));
     }
+
+    /**
+     * @Route("/search", name="blog_search")
+     * @Method("GET")
+     *
+     * @return JsonResponse
+     */
+    public function searchAction(Request $request)
+    {
+        $q = $request->query->get('q');
```

Comment:
It's vastly using name for a search request. The Symfony search uses this name http://api.symfony.com/2.7/index.html?q=request